Versions based on balenalib image fail to build on AMD64

Building of Dockerfiles based on `balenalib` images (2.4, 2.5) fails on AMD64 hosts:
```
$ make build version=2.5 arch=arm
-e [zeiot] build zeiot/rpi-prometheus:v2.5.0-arm
Sending build context to Docker daemon  7.168kB
Step 1/12 : FROM balenalib/armv7hf-debian:stretch
# Executing 1 build trigger

 ---> Running in 64a60f56ce48
standard_init_linux.go:190: exec user process caused "no such file or directory"
The command '/bin/sh -c cat /.balena/image-info' returned a non-zero code: 1
Makefile:46: recipe for target 'build' failed
make: *** [build] Error 1
```

This is most likely caused by an `ONBUILD` statement in `balenalib` image which triggers execution of `cat` binary built for ARM architecture even on hosts with non-ARM architectures:

```
$ docker image inspect balenalib/armv7hf-debian:stretch | grep OnBuild -A 2 
            "OnBuild": [
                "RUN cat /.balena/image-info"
            ],
[...]
```

It seems this bug does not affect `resin/*` images, so a fix could be to switch back to use them instead of `balenalib`.

--- a/2.4/Dockerfile.arm
+++ b/2.4/Dockerfile.arm
@@ -16,7 +16,7 @@
 # AUTHOR:         Nicolas Lamirault <nicolas.lamirault@gmail.com>
 # DESCRIPTION:    zeiot/rpi-prometheus
 
-FROM balenalib/armv7hf-debian:stretch
+FROM resin/armv7hf-debian:stretch
 
 LABEL summary="Prometheus for ARM" \
       description="Prometheus for ARM devices" \

--- a/2.4/Dockerfile.arm64
+++ b/2.4/Dockerfile.arm64
@@ -16,7 +16,7 @@
 # AUTHOR:         Nicolas Lamirault <nicolas.lamirault@gmail.com>
 # DESCRIPTION:    zeiot/rpi-prometheus
 
-FROM balenalib/aarch64-debian:stretch
+FROM resin/aarch64-debian:stretch
 
 LABEL summary="Prometheus for ARM" \
       description="Prometheus for ARM devices" \

--- a/2.5/Dockerfile.arm
+++ b/2.5/Dockerfile.arm
@@ -16,7 +16,7 @@
 # AUTHOR:         Nicolas Lamirault <nicolas.lamirault@gmail.com>
 # DESCRIPTION:    zeiot/rpi-prometheus
 
-FROM balenalib/armv7hf-debian:stretch
+FROM resin/armv7hf-debian:stretch
 
 LABEL summary="Prometheus for ARM" \
       description="Prometheus for ARM devices" \

--- a/2.5/Dockerfile.arm64
+++ b/2.5/Dockerfile.arm64
@@ -16,7 +16,7 @@
 # AUTHOR:         Nicolas Lamirault <nicolas.lamirault@gmail.com>
 # DESCRIPTION:    zeiot/rpi-prometheus
 
-FROM balenalib/aarch64-debian:stretch
+FROM resin/aarch64-debian:stretch
 
 LABEL summary="Prometheus for ARM" \
       description="Prometheus for ARM devices" \

